frontend/src/machines/billingMachine.ts

```diff
@@ -1,3 +1,137 @@
-import { createMachine } from "xstate";
+import { createMachine, assign } from "xstate";
 
-export const billingMachine = createMachine({});
+type Billing = {
+  id: string;
+  title: string;
+  amount: number;
+};
+
+interface Context {
+  billings: Billing[];
+  isEditingForm: boolean;
+  isCreatingForm: boolean;
+}
+
+const billingMachine = createMachine<Context>({
+  id: "billing",
+  initial: "idle",
+  context: {
+    billings: [],
+    isCreatingForm: false,
+    isEditingForm: false,
```

Comment:
Mau tanya, ini kok gak ada context buat nampung value dari formnya ya ?

Reply:
Will handle it on the next PR ya